### PR TITLE
Add org.gnome.OfficeRunner

### PR DESCRIPTION
**Pull request by hadess**

### CODE REVIEW

File: org.gnome.OfficeRunner.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "modules": [
+        {
+            "name": "office-runner",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://git.gnome.org/office-runner"
```
Comment (nedrichards) [inline org.gnome.OfficeRunner.json]: this needs to be a reproducible source. so a git tag+commit or a release archive to be available on flathub.

Comment (hadess) [reply org.gnome.OfficeRunner.json]: That's done. It would have been replaced by a release once it got merged.

**Comment by nedrichards** [comment]
bot, build org.gnome.OfficeRunner

**Comment by nedrichards** [comment]
The appdata needs [OARS](https://hughsie.github.io/oars/) and release tags.

https://gitlab.gnome.org/GNOME/office-runner/blob/master/src/office-runner.appdata.xml.in

**Comment by nedrichards** [comment]
Builds failed: https://flathub.org/builds/#/builders/2/builds/2402 the appstream didn't compose properly (usually to do with not being able to link up the icons, desktop files and appdata, could be the launchable being wrong, that sort of thing)

**Comment by hadess** [comment]
I added an empty OARS field and licences.

**Comment by hadess** [comment]
Looks like it failed again in https://flathub.org/builds/#/builders/24/builds/255
```
stat: cannot stat 'builddir/*/share/app-info/xmls/org.gnome.OfficeRunner.xml.gz': No such file or directory
```

I have no idea what's supposed to be in this file. Whatever the bot is supposed to be doing here, an error earlier when the file fails to be generated (?) would be useful.

**Comment by nedrichards** [comment]
This is "didn't export an appstream file properly" I gave my best guesses
earlier. Will try and take a minute and check out the branch and build to
see if I can offer anything more.

**Comment by hadess** [comment]
The problem is that the file gets installed as $prefix/share/appdata/office-runner.appdata.xml. The error handling really needs to be better :/

I've fixed this in the upstream repo.

**Comment by nedrichards** [comment]
bot, build org.gnome.OfficeRunner

**Comment by nedrichards** [comment]
Agreed on the error handling.

**Comment by nedrichards** [comment]
bot, build org.gnome.OfficeRunner

**Comment by nedrichards** [comment]
The id in your desktop file is for the old desktop file:

  <id type="desktop">office-runner.desktop</id>

Is what:

>  argv: zgrep "<id>org.gnome.OfficeRunner\(.desktop\)\?</id>" builddir/*/share/app-info/xmls/org.gnome.OfficeRunner.xml.gz

means. Whilst building locally to test I also saw that you weren't exporting any icons - so renamed them in the manifest. It'd be nicer to have them renamed upstream obviously.

**Comment by hadess** [comment]
> The id in your desktop file is for the old desktop file:

Fixed upstream.

> It'd be nicer to have them renamed upstream obviously.

That's done upstream as well.

Note that the build logs don't get propagated here, so I have no visibility over the errors at all, and when I do, the errors just aren't descriptive.

**Comment by nedrichards** [comment]
bot, build org.gnome.OfficeRunner

**Comment by nedrichards** [comment]
The builds not being pushed here reliably is deeply irritating. Everything is available on https://flathub.org/builds but agreed that it's not really integrated or helpful.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.OfficeRunner

**Comment by hadess** [comment]
> The builds not being pushed here reliably is deeply irritating. Everything is available on https://flathub.org/builds but agreed that it's not really integrated or helpful.

I filed https://github.com/flathub/buildbot-config/issues/35 about this. And about the errors themselves: https://github.com/flathub/buildbot-config/issues/34

Thanks for your patience :)

**Pull request closed without merging**